src/wfgoto.cpp

```diff
@@ -162,29 +157,27 @@ vector<PDNODE> *TreeIntersection(vector<vector<PDNODE> *> trees)
 		if (i == 1)
 		{
 			// on first time through loop, output is A and 'first' is trees[0];
-			first = trees[0];
+			first = &trees[0];

```

Comment:
Consider using `.data()` instead if you can enable c++17 mode and `/permissive-`

Reply:
I think "address of first element" expresses the intent better than .data(), even though it's the same thing